plover/gui_qt/machine_options.py

```diff
@@ -13,14 +14,17 @@
 
 _ = get_gettext()
 
-
```

Comment:
It's best to not make unnecessary style changes.

Reply:
fixed in 7c262f88d161f70e3017bf86d5c91d6f593e2e38